build.gradle

```diff
@@ -82,6 +82,16 @@ subprojects {
   apply plugin: "com.github.spotbugs"
   apply plugin: 'jacoco'
 
+
+  dependencies {
+    constraints {
```

Comment:
I am curious why we preferred to put this constraint under `subprojects` rather than individual `project`?

Reply:
Actually you're right.  I just reviewed and find that we can cope only in the `cruise-control` project.
Fixed

![Screen Shot 2022-03-31 at 11 10 48 AM](https://user-images.githubusercontent.com/935681/161122015-1d36ef85-ec67-49ec-9af9-09967e49cc65.png)